docs/mainpage.md

```diff
@@ -1,28 +1,28 @@
 
-Additionally the SDK provides higher-level libraries for dealing with timers, USB, synchronization and multi-core programming, along with additional high-level functionality built using PIO, such as audio. The SDK can be used to build anything from simple applications, or full-fledged runtime environments such as MicroPython, to low-level software such as the microcontroller's on-chip bootrom itself.
+Additionally, the SDK provides higher-level libraries for dealing with timers, USB, synchronisation, and multi-core programming, along with high-level functionality built using PIO (such as audio). The SDK can be used to build anything from simple applications to full-fledged runtime environments such as MicroPython, or even low-level software, such as the microcontroller's on-chip boot ROM.
 
-This documentation is generated from the SDK source tree using Doxygen. It provides basic information on the APIs used for each library, but does not provide usage information. Please refer to the Databooks for usage and more technical information.
+This documentation is generated from the SDK source tree using Doxygen. It provides basic information on the APIs used for each library, but doesn't provide usage information. For more technical information and usage guidance, refer to the Raspberry Pi databooks.
 
 ## SDK Design
 
-The RP-series microcontroller range are powerful chips, however they are used in an embedded environment, so both RAM and program space are at premium. Additionally the trade-offs between performance and other factors (e.g. edge-case error handling, runtime vs compile-time configuration) are necessarily much more visible to the developer than they might be on other higher-level platforms.
+The RP-series microcontrollers are powerful chips designed for embedded systems, which means that these chips operate in environments with limited memory (RAM) and storage (program space). Developers must balance performance with other factors, such as edge-case error handling, runtime versus compile time, and so on. These trade-offs are necessarily more visible to developers for RP-series microcontrollers than they might be for higher-level platforms (like desktop systems), where such considerations are often abstracted or automated.
 
-The intention within the SDK has been for features to just work out of the box, with sensible defaults, but also to give the developer as much control and power as possible (if they want it) to fine-tune every aspect of the application they are building and the libraries used.
+The SDK is designed to be both beginner friendly and powerful for more experienced users. Its features work out-of-the-box, with sensible defaults that cover most use cases. At the same time, it gives developers as much control as possible to adjust the application they're building and the libraries used, if they choose to.
```

Comment:
this "adjust", and indeed the loss "fine-tune every aspect of" loses the original meaning of this sentence. as said, you can optionally fine-tune every aspect, "adjust" is meanigless

Reply:
we give them lots of knobs to twiddle if they really care